src/pkg/control/options.go

```diff
@@ -13,6 +13,7 @@ type Options struct {
 	RestorePermissions   bool            `json:"restorePermissions"`
 	SkipReduce           bool            `json:"skipReduce"`
 	ToggleFeatures       Toggles         `json:"ToggleFeatures"`
+	SkipMigrations       bool            `json:"skipMigrations"`
```

Comment:
nit: as Options is exported I would prefer inverting this flag so that even if someone uses just `control.Options{}` we don't accidentally enable code paths we don't want to

Reply:
also this may be more suited as a toggle feature? I assume it's temporary for right now

Reply:
sounds good